pokedex/src/components/Trainers/EditTrainersModal.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+const { useState, useEffect } = React;
+// TODO: Fix devolve and evolve later
+
+export default function EditModal({ show, handleClose, pokemonInfo, handleSave, pokemonInfoIndex }) {
+    const [nationalNumber, setNationalNumber] = useState("")
+    const [name, setName] = useState("")
+    const [photoUrl, setPhotoUrl] = useState("")
+    const [description, setDescription] = useState("")
+    // const [devolvedState, setDevolvedState] = useState("")
```

Comment:
these three should be what the trainer has - see here https://lucid.app/lucidchart/f98bafbe-6c22-476e-a849-04b3d1db9f31/edit?invitationId=inv_1a1d07ae-4490-42e8-9900-0ca6f5c7bbbb&page=0_0#